impeller/display_list/skia_conversions.h

```diff
@@ -24,6 +24,10 @@
 namespace impeller {
 namespace skia_conversions {
 
+/// @brief Like SkRRect.isSimple, but allows the corners to differ by
```

Comment:
Please define "simple", skia doesn't have a docstring for it.  Something like "True if independently each corner curves in almost equal amounts in the x and y direction."

Reply:
Done

Reply:
Unfortunately too limited (and agreed that Skia didn't document this well, but code inspection and tests show the truth). Simple means the corners are all the same *pair* of numbers. The corners of a simple RRect are elliptical, not circular. So, the test here is too limiting. I'll file an issue to relax it back to the original definition, but with relaxed comparisons - all X radii are equal and all Y radii are equal and the two sets can be different.

Reply:
See https://github.com/flutter/flutter/issues/151034